ibtk/include/ibtk/PatchMathOps.h

```diff
@@ -226,6 +226,21 @@ class PatchMathOps : public SAMRAI::tbox::DescribedClass
              int l = 0,
              int m = 0) const;
 
+    /*!
+     * \brief Computes dst_l = alpha div (coef 1 src1) + beta src2_m.
+     *
+     * Uses centered differences.
+     */
+    void div(SAMRAI::tbox::Pointer<SAMRAI::pdat::CellData<NDIM, double> > dst,
+             double alpha,
+             SAMRAI::tbox::Pointer<SAMRAI::pdat::SideData<NDIM, double> > src1,
```

Comment:
Same comments as above ---
1. The convention for most of these operators is that "coefficients" come before the "variable" that they are hitting. Can we stick with that convention here?
2. Can we merge together `alpha` and `coef1` so that there is just one coefficient per variable?